web/templates/pages/privacy.html

```diff
@@ -0,0 +1,16 @@
+{% extends 'base.html' %}
+
+{% block title %}Privacy Policy{% endblock %}
+
+{% block content%}
+
+	<div>
+		<h3 class="privacy">TrackCat Privacy Policy</h3>
+		<p>When you register for TrackCat we ask for information such as your name, GitHub account and email address. The information we collect is used to improve the content of our web pages and the quality of our service, and is not shared with or sold to other organizations for commercial purposes.</p>
+		<p>TrackCat uses collected information for the following general purposes: identification and authentication, ful acces to our services, contact, and research.</p>
+		<p>It is necessary to share information in order to investigate, prevent, or take action regarding illegal activities, suspected fraud, situations involving potential threats to the physical safety of any person, or as otherwise required by law.</p>
+
+		<p>TrackCat may periodically update this policy. We will notify you about significant changes in the way we treat personal information by sending a notice to the email address specified in your TrackCat account or by placing a prominent notice on our site.</p>
+	</div>
+
+{% endblock content%}
```

Comment:
please add a newline at the end of the file (go to the end of this line and just press enter)